NanoAnalysis/python/modules/jetJERC.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+def getJetCorrected(era, tag, is_mc, overwritePt=True) :
+    from PhysicsTools.NATModules.modules.jetCorr import jetJERC
+
+    if era != 2022:
+        raise ValueError("getJetCorrected: Era", era, "not supported")
+
+    if is_mc :
+        if tag == "pre_EE":
+            folderKey = "2022_Summer22"
+            L1Key = "Summer22_22Sep2023_V2_MC_L1FastJet_AK4PFPuppi"
+            L2Key = "Summer22_22Sep2023_V2_MC_L2Relative_AK4PFPuppi"
+            L3Key = "Summer22_22Sep2023_V2_MC_L3Absolute_AK4PFPuppi"
+            L2L3Key = "Summer22_22Sep2023_V2_MC_L2L3Residual_AK4PFPuppi"
+            smearKey = "JERSmear"
+            JERKey = "Summer22_22Sep2023_JRV1_MC_PtResolution_AK4PFPuppi"
+            JERsfKey = "Summer22_22Sep2023_JRV1_MC_ScaleFactor_AK4PFPuppi"
+        elif tag == "post_EE":
```

Comment:
also I would keep this as the default case (`else `instead than `elif`), to catch anything that is not pre_EE.
(this is the current behaviour of other modules that distinguish pre and post EE)

Reply:
Done